Tests/SnapshotConfiguration.swift

```diff
@@ -22,6 +22,9 @@ struct SnapshotConfiguration {
   /// A custom `AnimationImageProvider` to use when rendering this animation
   var customImageProvider: AnimationImageProvider?
 
+  /// A custom `AnimationTextProvider` to use when rendering this animatino
+  var customTextProvider: AnimationTextProvider?
```

Comment:
Make sure to actually apply this text provider to the `AnimationView` in `SnapshotConfiguration.makeAnimationView`: https://github.com/airbnb/lottie-ios/blob/master/Tests/SnapshotTests.swift#L255

Reply:
Cool to know, thanks!
Done in [c7a21ce](https://github.com/airbnb/lottie-ios/pull/1723/commits/c7a21ce845e9fbd593a2ab8f22c2ffae569ef163) ✅